Makefile

```diff
@@ -88,6 +88,7 @@ D_SRCS += debug/sbuf.c
 C_SRCS += debug/printf.c
 D_SRCS += boot/acpi.c
```

Comment:
you probably need to do code rebase, it conflicts with https://github.com/projectacrn/acrn-hypervisor/commit/e8236e552d8aa07953cb728a78f1dff9903d9c9f